tools/cpp/osx_cc_wrapper.sh.tpl

```diff
@@ -110,7 +110,7 @@ for rpath in ${RPATHS}; do
         if [[ -n "${libname-}" ]]; then
             libpath=$(get_library_path ${lib})
             if [ -n "${libpath}" ]; then
-                ${INSTALL_NAME_TOOL} -change $(get_otool_path "${libpath}") \
+                "${INSTALL_NAME_TOOL}" -change $(get_otool_path "${libpath}") \
```

Comment:
Maybe just in-line this? With this currently I think your quotes are off?

Reply:
Thanks